prismic-model/articles.json

```diff
@@ -69,6 +73,33 @@
               }
             }
           },
+          "gifVideo" : {
+            "type" : "Slice",
+            "fieldset" : "Gif video",
+            "non-repeat" : {
+              "caption" : {
+                "type" : "StructuredText",
+                "config" : {
+                  "single" : "hyperlink, bold, em",
+                  "label" : "Caption"
+                }
+              },
+              "posterFrame" : {
+                "type" : "Image",
+                "config" : {
+                  "label" : "Poster frame"
+                }
+              },
+              "movieFile" : {
```

Comment:
Could we use videoFile (and elsewhere where we've used `movie`?) - it's more consistent with HTML etc.

Reply:
We never really post-fix things with `File` - see `audio` and `image`.